app/models/manageiq/providers/vmware/infra_manager.rb

```diff
@@ -399,6 +399,11 @@ def vm_acquire_mks_ticket(vm, options = {})
     end
     alias_method :vm_remote_console_mks_acquire_ticket, :vm_acquire_mks_ticket
 
+    def vm_acquire_ticket(vm, options = {})
+      invoke_vim_ws(:acquireTicket, vm, options[:user_event], options[:ticket_type])
+    end
+    alias_method :vm_remote_console_acquire_ticket, :vm_acquire_ticket
```

Comment:
Okay I added a method to get webmks tickets specifically so you don't need to pass a `:ticket_type` in options